approach.md

```diff
@@ -43,7 +43,7 @@ This should include the collaborative creation, adaptation and translation of co
 
 > Allowing the discovery, tracking and updating of content used from other sources is a design goal.
 
-Multimedia content, internal links or external links should be easy to navigate and include in content.
+Multimedia content, internal and external links should be easy to navigate and include in new content.
```

Comment:
Agreed with the simpler approach to internal and external but I think all of these type of content can be included in new and existing content not only new content!